### PR TITLE
Update winbase.d

### PR DESCRIPTION
**Pull request by etcimon**
Add missing symbols

### CODE REVIEW

**Comment by etcimon** [comment]
I get this error when calling, any idea?

```
unction core.sys.windows.winbase.GetTokenInformation(void*, TOKEN_INFORMATION_CLASS, void*, uint, uint*) is not callable using argument types 
(void*, int, void*, uint, uint*) source\winhelpers.d(292,27):        cannot pass argument TokenElevationType of type int to parameter TOKEN_INFORMATION_CLASS

	if (!GetTokenInformation(hToken, TokenElevationType, cast(void*)&elevType, cast(uint)elevType.sizeof, cast(uint*)&cbSize))
```

**Comment by MoonlightSentinel** [comment]
Named enums build a new type ([spec](https://dlang.org/spec/enum.html#named_enums)). Seems like you tried to pass an `int` instead of a member of `TOKEN_INFORMATION_CLASS`

File: src/core/sys/windows/winbase.d [hunk 1]
```diff
@@ -2257,7 +2257,8 @@ WINBASEAPI BOOL WINAPI SetEvent(HANDLE);
     BOOL IsTextUnicode(PCVOID, int, LPINT);
     BOOL IsValidAcl(PACL);
     BOOL IsValidSecurityDescriptor(PSECURITY_DESCRIPTOR);
-    BOOL IsValidSid(PSID);
+    BOOL IsValidSid(PSID);    
+	BOOL CreateWellKnownSid(WELL_KNOWN_SID_TYPE, PSID, PSID, PDWORD);
```
Comment (MoonlightSentinel) [inline src/core/sys/windows/winbase.d]: `WELL_KNOWN_SID_TYPE` is  missing

Comment (MoonlightSentinel) [reply src/core/sys/windows/winbase.d]: Fix intendation

File: src/core/sys/windows/winbase.d [hunk 2]
```diff
@@ -2257,7 +2257,8 @@ WINBASEAPI BOOL WINAPI SetEvent(HANDLE);
     BOOL IsTextUnicode(PCVOID, int, LPINT);
     BOOL IsValidAcl(PACL);
     BOOL IsValidSecurityDescriptor(PSECURITY_DESCRIPTOR);
-    BOOL IsValidSid(PSID);
+    BOOL IsValidSid(PSID);
```
Comment (MoonlightSentinel) [inline src/core/sys/windows/winbase.d]: Trailing space

File: src/core/sys/windows/winbase.d (hunk 1 above)
Comment (etcimon) [reply src/core/sys/windows/winbase.d]: The type comes from the changes in winnt.d

Comment (MoonlightSentinel) [reply src/core/sys/windows/winbase.d]: I see. It's usually a good idea to reference required PRs in the description.

**Pull request closed without merging**